codecov.yaml

```diff
@@ -0,0 +1,193 @@
+codecov:
```

Comment:
> Generally if your uploaded reports fail to process, path fixing may help. You should download the coverage report generated within your CI for a particular commit, examine the file paths located within it, and see if they map directly to your git/hg file structure. If not, you can attempt to apply a path fix and see if this helps on subsequent commits.

Do you know where I can download the coverage report?
https://docs.codecov.com/docs/fixing-paths#how-do-i-know-if-i-should-use-path-fixing

Reply:
here https://app.codecov.io/gh/autowarefoundation/autoware.universe/commit/5af18dcbe100f7cd305c0be1e445d9e46bb8e1c7
![image](https://user-images.githubusercontent.com/19993104/153329820-51e61333-50ce-4a2c-8b4b-acee2af47f60.png)

Strangely, we can access the source files from commit, branch and pull request views.
Overview is still not working.

Reply:
Thank you.
Then, we need more investigation.

Reply:
Yes, perhaps additional settings are needed.

Reply:
> - <files in coverage report>::<corresponding paths in git/hg project>

Could you write here some concrete examples of what are the `<files in coverage report>` and what are the `<corresponding paths in git/hg project>`?

Reply:
In my point of view, `<files in coverage report>` is

```
SF:/__w/autoware.universe/autoware.universe/planning/behavior_velocity_planner/include/scene_module/blind_spot/manager.hpp
```
(SF stands for source file. https://ja.osdn.net/projects/lcov-jp/wiki/geninfo.1 )

from
https://api.codecov.io/upload/gh/autowarefoundation/autoware.universe/download?path=v4/raw/2022-02-10/EA9EE84B5FC56CCEF5197BE5F31E99EC/eb599aec607084113cf0994888031ebe6065468a/ca733788-b72e-4a3d-b24a-9fd37af6e8cd.txt

`<corresponding paths in git/hg project>` is
![image](https://user-images.githubusercontent.com/19993104/153342988-5c078fa9-13c9-4d6c-821f-40cd2b44ac0f.png)

from https://app.codecov.io/gh/autowarefoundation/autoware.universe